src/Licenses/Entities/License/Prediction.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Licenses\Entities\License;
+
+use ArrowSphere\PublicApiClient\AbstractEntity;
+use ArrowSphere\PublicApiClient\Exception\EntityValidationException;
+
+/**
+ * Class ActiveSeats
+ */
+class Prediction extends AbstractEntity
```

Comment:
I think this class should be named "Predictions" or "PredictionResponse" and the other one should be named "Prediction"
```suggestion
class PredictionResponse extends AbstractEntity
```